cpp/arcticdb/version/local_versioned_engine.cpp

```diff
@@ -1768,20 +1768,61 @@ timestamp LocalVersionedEngine::latest_timestamp(const std::string& symbol) {
 }
 
 std::unordered_map<KeyType, std::pair<size_t, size_t>> LocalVersionedEngine::scan_object_sizes() {
-    std::unordered_map<KeyType, std::pair<size_t, size_t>> sizes;
+    std::unordered_map<KeyType, std::pair<size_t, std::atomic<size_t>>> sizes;
```

Comment:
I might be lacking some context. What should the values in `std::pair` above represent?